app/scripts/controllers/transactions/tx-state-manager.js

```diff
@@ -128,7 +130,7 @@ export default class TransactionStateManager extends EventEmitter {
       id: createId(),
       time: new Date().getTime(),
       status: TransactionStatus.unapproved,
-      metamaskNetworkId: netId,
+      metamaskNetworkId: hexToBn(chainId).toString(10),
```

Comment:
This may not be correct for all networks, but removing this property entirely would be a larger effort.

Reply:
Hmm, I thought we had discussed the possibility of submitting a PR ahead of this one that would stop setting this property. Has that been considered? It doesn't seem safe to set invalid network IDs like this.

At first glance it doesn't seem like it would be that complex. It's only being set on 3 lines in this repo.

Reply:
This is now reverted to use network ID.